apps/api/core/storage.py

```
@@ -6,21 +6,27 @@
             endpoint_url=endpoint_url,
-            aws_access_key_id=settings.R2_ACCESS_KEY_ID,
-            aws_secret_access_key=settings.R2_SECRET_ACCESS_KEY,
+            aws_access_key_id=getattr(settings, 'R2_ACCESS_KEY_ID', ''),
+            aws_secret_access_key=getattr(settings, 'R2_SECRET_ACCESS_KEY', ''),
             region_name='auto'  # R2 requires region to be 'auto'
         )
 
+        # Ensure bucket exists (for MinIO local development)
+        try:
+            self.s3_client.head_bucket(Bucket=self.bucket)
+        except Exception:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)